pkg/testutil/nerdtest/utilities.go

```diff
@@ -25,15 +25,19 @@ import (
 
 	"github.com/containerd/nerdctl/v2/pkg/inspecttypes/dockercompat"
 	"github.com/containerd/nerdctl/v2/pkg/inspecttypes/native"
+	"github.com/containerd/nerdctl/v2/pkg/testutil"
 	"github.com/containerd/nerdctl/v2/pkg/testutil/test"
 )
 
+func IsDocker() bool {
+	return testutil.GetTarget() == "docker"
+}
```

Comment:
Why not just call testutil.GetTarget directly

Reply:
Migrating to the proposed new framework, I would rather have everything sourced from there to help cleaning things up.
And yes, eventually, they will be merged.